tests/std/tests/LWG3422_seed_seq_ctors/test.compile.pass.cpp

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <random>
+#include <type_traits>
+#include <vector>
+
+using namespace std;
+
+int main() {
+    static_assert(is_nothrow_default_constructible_v<seed_seq>, "");
+    static_assert(!is_constructible_v<seed_seq, initializer_list<vector<int>::iterator>>, "");

```

Comment:
Maybe add positive test case too?
```suggestion
    static_assert(!is_constructible_v<seed_seq, initializer_list<vector<int>::iterator>>, "");
    static_assert(is_constructible_v<seed_seq, initializer_list<int>, "");
```